1 table.sql

```diff
@@ -0,0 +1,18 @@
+drop TABLE if EXISTS dbo.CleaningCompany
+go
+CREATE table dbo.CleaningCompany(
+        CustomerID int not null IDENTITY primary key,
+        FirstName varchar(50) not null constraint ck_Customer_First_name_is_not_blank CHECK (FirstName <> ''), 
+        LastName varchar(50) not null constraint ck_Customer_Last_name_is_not_blank CHECK (LastName <> ''), 
+        PhoneNumber varchar(15) not null constraint ck_Customer_Phone_Number_is_not_blank CHECK (PhoneNumber <> ''), 
+        CustomerAddress VARCHAR(50) not null constraint ck_Customer_Address_is_not_blank CHECK (CustomerAddress <> ''), 
```

Comment:
Address should be unique. As you can have multiple times the same customer but in different addresses.

Reply:
added unique contraint